test_cromwell.sh

```diff
@@ -120,8 +120,12 @@ sbt test:compile
 CP=$(sbt "export test:dependency-classpath" --error)
 
 if [[ -n ${EXCLUDE_TAG} ]]; then
-    echo "Running Centaur filtering out ${EXCLUDE_TAG} tests"
-    TEST_COMMAND="java ${REFRESH_TOKEN} -cp $CP org.scalatest.tools.Runner -R target/scala-2.12/test-classes -oD -PS${TEST_THREAD_COUNT} -l ${EXCLUDE_TAG}"
+    echo "Running Centaur filtering out ${EXCLUDE_TAG[@]} tests"
```

Comment:
We try to make sure shellcheck doesn't complain (despite there being some preexisting complaints), this line runs afoul of this: https://github.com/koalaman/shellcheck/wiki/SC2145